docs/contributing/hack.md

```diff
@@ -30,6 +69,12 @@ Ready to contribute? Here's how to set up *Watson* for local development.
 6. If you have added a new command or updated/fixed docstrings, please update the documentation:
 
         $ make docs
+   
+     Beware that mkdocs 0.14 is not compatible with Python 3.7: you should either
+     use a virtualenv with Python 3.6 or update `mkdocs` to a more recent version
+     (1.0.4 at the time of writing, which also requires running `pip install
+     --upgrade mkdocs-bootstrap mkdocs-bootswatch` to get the `flatly` theme
+     used by our docs). 
```

Comment:
I think we should rather update our requirements so that we don't install `mkdocs` <= 0.14.

Reply:
Yes, but it should be handled in a separated PR @raphaelfournier :wink:

Reply:
Added issue #280 to handle this.